eva/catalog/catalog_manager.py

```diff
@@ -322,191 +384,90 @@ def udf_io(
             dimensions(List[int]):dimensions of the io created
             is_input(bool): whether a input or output, if true it is an input
         """
-        return UdfIO(
+        return UdfIOCatalog(
             io_name,
             data_type,
             array_type=array_type,
             array_dimensions=dimensions,
             is_input=is_input,
         )
 
-    def create_udf(
-        self,
-        name: str,
-        impl_file_path: str,
-        type: str,
-        udf_io_list: List[UdfIO],
-    ) -> UdfMetadata:
-        """Creates an udf metadata object and udf_io objects and persists them
-        in database.
-
-        Arguments:
-            name(str): name of the udf to which this metdata corresponds
-            impl_file_path(str): implementation path of the udf,
-                                 relative to eva/udf
-            type(str): what kind of udf operator like classification,
-                                                        detection etc
-            udf_io_list(List[UdfIO]): input/output info of this udf
-
-        Returns:
-            The persisted UdfMetadata object with the id field populated.
-        """
-
-        metadata = self._udf_service.create_udf(name, impl_file_path, type)
-        for udf_io in udf_io_list:
-            udf_io.udf_id = metadata.id
-        self._udf_io_service.add_udf_io(udf_io_list)
-        return metadata
+    def create_multimedia_table_catalog_entry(
```

Comment:
Can you elaborate more diff between `create_multimedia_table_catalog_entry` and `create_media_metadata_table_catalog_entry `?